internal/ui/assets.go

```diff
@@ -0,0 +1,25 @@
+// +build ui
+
+package ui
+
+import (
+	"embed"
+	"io/fs"
+	"net/http"
+)
+
+// Sadly we can't embed this into the embed line, but we use it elsewhere
+const contentDir = ".tmp/boundary-ui/ui/admin/dist"
+
+// content is our static web server content.
+//go:embed .tmp/boundary-ui/ui/admin/dist
+var content embed.FS
+
+func Handler() http.Handler {
```

Comment:
Any desire to make this more testable via a func that just returns the desired embed FS?    That would allow us to assert at least some of the assets we believe should be available from the `dist` directory.

Reply:
Sure, I can do that.